docs/tutorials/intro.md

```diff
@@ -70,102 +72,72 @@ algokit init
 
-Once finished, (if you have it installed) VS Code should automatically be opened with the initialised project and you will be prompted to install appropriate VS Code extensions. This starter app will contain one smart contract (built using the [Beaker](https://beaker.algo.xyz/) smart contract development framework) named `helloworld.py`, in the `hello_world` folder, with one method (`hello`) that takes a `String` and returns a `String`.
+Once finished, (if you have it installed) VS Code should automatically be opened with the initialised project and you will be prompted to install appropriate VS Code extensions. This starter app will contain one smart contract (built using the [Algorand Python](https://algorandfoundation.github.io/puya/) language) named `contract.py`, in the `smart_contracts/hello_world` folder, with one method (`hello`) that takes a `String` and returns a `String`. Notice that within the contract folder, there is a `deploy_config.py`, this defined the parameters for deployment of your smart contract to the select target chain (Algorand TestNet, Algorand MainNet, Algorand LocalNet or else).
 
 ![AlgoKit Playground Contract](../imgs/algokitplayground.png)
 
-## Run the Demo Application
+## AlgoKit Project structure 🏗
 
-Once the playground project is created, you will notice in the `hello_world` folder a file named `demo.py` which is a simple example of using AlgoKit to deploy and make a call to the `helloworld.py` smart contract on the LocalNet instance started earlier.
+Since you have just instantiated an official AlgoKit template, each official smart contract template includes an interactive codespace walkthrough powered by [CodeTours](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour).
 
-![AlgoKit Playground Demo](../imgs/algokitdemo.png)
+To start the interactive walkthrough, install the extension and click on the green play button in the left bottom corner of the `Explorer` pane as demonstrated on the screenshot below (See `Getting Started with Your AlgoKit Project`).
 
-Right clicking on this file and selecting `Run Python File in Terminal` will deploy the `HelloWorldApp` smart contract and then call it passing the parameter `name` with a value of `Beaker`. Alternatively, you can hit F5 (or whatever keyboard shortcut is set in VS Code for running the debugger) while you are viewing the `helloworld.py` file and it will:
+![CodeTour Play Button](../imgs/algokitcodetour.png)
 
-1. Start LocalNet
-2. Build the smart contract
-3. Deploy and call the smart contract (`demo.py`)
+Upon execution, the walkthrough will guide you through the key components of your fresh AlgoKit project instance, which is an important prerequisite to learn before running and deploying your first smart contract. Certainly, not because its hard to learn, but because it will save you a lot of time and efforts if you set the right foundation first.
```

Comment:
```suggestion
Upon execution, the walkthrough will guide you through the key components of your fresh AlgoKit project, which is an important prerequisite to learn before running and deploying your first smart contract. Certainly, not because its hard to learn, but because it will save you a lot of time and effort having the correct foundations.
```